language/test/Language_test.rb

```diff
@@ -160,4 +161,17 @@
     end
   end
 
+  describe "#GetLocaleString" do
+    context "when the language is defined" do
+      it "returns the full language identifier" do
+        expect(subject.GetLocaleString("de_ZU")).to eq("de_ZU.UTF-8")
+      end
+    end
+
+    context "when the language is not defined" do
+      it "returns the full language identifier" do
+        expect(subject.GetLocaleString("ma_MA")).to eq("ma_MA.UTF-8")
+      end
+    end
+  end
```

Comment:
Because it's your own PR :) I'll extend the text to use a different `@use_utf8` value and it will make it more clear.